packages/hardhat-ethers/src/internal/websocket.ts

```diff
@@ -0,0 +1,39 @@
+import type EthersT from "ethers";
+import { extendEnvironment } from "hardhat/config";
+import { lazyObject } from "hardhat/plugins";
+
+import {
+  getContractAt,
+  getContractFactory,
+  getSigner,
+  getSigners,
+} from "./helpers";
+import type * as ProviderProxyT from "./ethers-websocket-provider-proxy";
+import "./type-extensions";
+
+extendEnvironment((hre) => {
```

Comment:
Is this meant to be an alternative version of the plugin?

Reply:
No? haha. I just hacked my way through getting the test to work.  I'm not even sure exactly what it is I did. :sweat_smile:

Reply:
Eliminated this in https://github.com/nomiclabs/hardhat/pull/1792/commits/3ec7a887d380aaaa1e7e119ad15dcc96d94d3a82